ovn_k8s/watcher/watcher.py

```diff
@@ -23,6 +27,29 @@
 from ovn_k8s.processor import conn_processor
 
 vlog = ovs.vlog.Vlog("watcher")
+exiting = False
+
+
+def _unixctl_exit(conn, unused_argv, unused_aux):
+    global exiting
+    exiting = True
+    conn.reply(None)
+
+
+def _unixctl_run():
+    ovs.unixctl.command_register("exit", "", 0, 0, _unixctl_exit, None)
+    error, unixctl_server = ovs.unixctl.server.UnixctlServer.create(None)
+    if error:
+        ovs.util.ovs_fatal(error, "could not create unixctl server", vlog)
+
+    while True:
+        unixctl_server.run()
+        if exiting:
+            unixctl_server.close()
+            sys.exit()
```

Comment:
What are the intended semantics of ovs-appctl exit?
This causes an abrupt shutdown, which might kill the processor while in the middle of configuring a logical port.

I see implement a graceful shutdown mechanism as not trivial, so maybe we can leave that for another PR.